content/kubernetes/deployment/quick-start.md

```diff
@@ -147,6 +147,7 @@ You can test the operator by creating a minimal cluster by following this proced
           cpu: 2000m
           memory: 16Gi
     ```
+    {{%note%}}Each cluster must have at least 3 nodes. One node RECs are not supported.{{%/note%}}
```

Comment:
There's a formatting issue in the next line where it's treating the sentence as code. I think changing the tabs might fix this as well.
```suggestion
    {{%note%}}
Each cluster must have at least 3 nodes. One-node RECs are not supported.
    {{%/note%}}
```